src/components/DiagramFrame/DiagramCanvas/DiagramToolbar.tsx

```diff
@@ -32,6 +32,9 @@ import {
   IconButton,
   theme
 } from "@looker/components"
+import { CenterFocusWeak, Map } from "@styled-icons/material"
+import { Plus, Minus } from "@styled-icons/boxicons-regular"
```

Comment:
https://docs.google.com/spreadsheets/d/1J6S6a9cxH0O9wVaDtsEKXbusOf2vhFDFMU0uaK70U9c/edit#gid=0

I would expect Minus to be Remove from material
I would expect Plus to be Add from material